### DISCUSSION TITLE
is it possible to run frigate as standalone and use app like pushover for notifications/snapshots?

### DISCUSSION THREAD

**Discussion by heassifilts** [blakeblackshear/frigate#6252] [Ask A Question] [answered]
Before I get started I want to say thank you to the developer of frigate for the work they put into this software and for the community here whose searchable posts helped me with installation immensely.

Right now I have a basic frigate docker install on an old laptop running openmediavault OS.  I saw the github thread here that mentioned that frigate works fine as a standalone and I’m content for now using the web interface and the yaml files to interact with frigate, but I’m a little confused what my options are if i want to start getting notifications on my iphone with a snapshot of the detected object.  I know the common method is to integrate frigate into home assistant and have Home Assistant do the notifications but are there other alternatives?   I can certainly see the appeal of this integration if a person is already using HA but for someone who isn’t using it it seems like an added complexity that could be bypassed with a different solution?  I see there are apps like pushover that allow for various push triggers..

is there a recommended HA setup if a person only intends on using it for frigate notifications and snap shots?  could the docker version of HA be run on the same server as frigate if HA is only used?

it’s also completely possible that i am overthinking this and HA is a simple program if i’m only going to be using it for notifications.  please let me know if my concern for the added complexity is unwarranted.

The searches I’ve found regarding these topics seem to focus on use cases where HA is already deployed in the home or will be setup to do more than interact with frigate.  I know there are other solutions than frigate for this like blue iris, zoneminder etc etc but i’d like to see if i can get frigate working for my needs.

thanks for the help

**Comment by NickM-27** [comment, marked as answer]
There are many options using other platforms like node_red. I have also seen some projects for integrating pushover and others into frigate. Frigate posts everything to MQTT so that any notification service can be used.

> is there a recommended HA setup if a person only intends on using it for frigate notifications and snap shots? could the docker version of HA be run on the same server as frigate if HA is only used?

yes

**Comment by heassifilts** [comment]
thank you for the prompt response and thank you for your contributions to the discussions.

I’ll dig into node_red and the pushover project and get more familiar with MQTT.  My rudimentary “frigate notifications without home assistant” like searches were leading to results about the HA blueprint not working etc so i’m glad I asked! 

I’ll post back when I figure out what path to go down.

**Comment by heassifilts** [comment]
marked as answered.  I finally had some time to sit down and familiarize myself more with mqtt and node-red and I’m happy to report I have setup a simple node-red configuration that sends me an email with the smaller sized snapshot image upon object detection in a zone i setup.

I may eventually swap out email for an app like pushover but for now i’ll be testing as is with email.   I may also try home assistant in the future to see what i’m missing out on but for now i’m having fun with a couple frigate instances testing different object models and fiddling with the knobs to reduce some false positives i have with some tree shadows…

thanks again
